lib/ruby_lsp/handler.rb

```diff
@@ -62,7 +62,20 @@ def handle(request)
             error = e
           end
 
-          @writer.write(id: request[:id], result: result) unless result == VOID
+          if error
+            @writer.write(
+              {
+                id: request[:id],
+                error: {
+                  code: LanguageServer::Protocol::Constant::ErrorCodes::INTERNAL_ERROR,
```

Comment:
We have the `LanguageServer::Protocol::Constant` assigned to `Constant`, so we can make this a little shorter and maybe even fit the hash into a single line.